{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_admin.py

```diff
@@ -47,3 +52,23 @@ def test_view_user(self, admin_client):
         url = reverse("admin:users_user_change", kwargs={"object_id": user.pk})
         response = admin_client.get(url)
         assert response.status_code == 200
+
+    @pytest.fixture
+    def force_allauth(self, settings):
+        settings.DJANGO_ADMIN_FORCE_ALLAUTH = True
+        # Reload the admin module to apply the setting change
+        import {{ cookiecutter.project_slug }}.users.admin as users_admin  # pylint: disable=import-outside-toplevel
+
+        try:
+            reload(users_admin)
+        except admin.sites.AlreadyRegistered:
+            pass
+
+    @pytest.mark.django_db
+    def test_allauth_login(self, rf, settings, force_allauth):  # pylint: disable=unused-argument
```

Comment:
Why do we need to pass the `settings` fixture? It's not directly used by this method, is it?

Reply:
Could probably also replace the `force_allauth` argument by using the decorator `@pytest.mark.use_fixtures("force_allauth")`, which would avoid the pylint issue.

Reply:
Thanks, the settings were a mistake and I didn't know about that marker.